docs/Resources/resources.md

```diff
@@ -26,13 +21,23 @@ More information on running Geppetto is [available online](http://geppetto.org).
 
 [The project page for Geppetto](/Projects/geppetto/) has information about getting involved in its development with OpenWorm.
 
+### Sibernetic
+
+Sibernetic is the code base that currently implements the crawling model. Sibernetic is a C++ / Python code base by Palyanov, Khayrulin and Vella that has been created expressly for the purpose of doing research and building the model quickly.
+
+![image](http://i.imgur.com/KSWjCaW.jpg)
+
+More information on running Sibernetic is [available online](http://sibernetic.org).
+
+[The project page for Sibernetic](/Projects/sibernetic/) has information about getting involved with its development.
+
 ### Connectome Engine and Lego Mindstorms robot
 
 To start getting some practical experience playing with dynamics that come from the connectome, we have simplified it into a project called the 'connectome engine' and integrated its dynamics into a Lego Mindstorms EV3 robot. You can [see a movie of this in action](https://www.youtube.com/watch?v=D8ogHHwqrkI).
 
 ![image](http://i.imgur.com/OG7sOAD.jpg)
 
-This is currently in the process of being written up.
+More information is available at [http://www.connectomeengine.org]
```

Comment:
i presume this change is ok?

Reply:
Appears to be a dead link. You can ask @Interintel if there's a better link/summary for this project now.